registry/configuring-registry-operator.adoc

```diff
@@ -4,6 +4,9 @@ include::modules/common-attributes.adoc[]
 :context: configuring-registry-operator
 toc::[]
 
+[id="image-registry-on-cloud"]
+== Image Registry on cloud platforms and OpenStack
+
 The Image Registry Operator installs a single instance of the {product-title}
 registry, and it manages all configuration of the registry, including setting up
```

Comment:
s/registry, and it manages all configuration of the registry, including/registry, and manages all registry configuration, including

Reply:
Updated.